docs/node/admin/kwil-admin/snapshot/create.mdx

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 1
+sidebar_label: "create"
+id: "kwil-admin-snapshot-create"
+title: "kwil-admin snapshot create"
+description: "Creates a snapshot of the database."
+slug: /ref/kwil-admin/snapshot/create
+---
+
+## kwil-admin snapshot create
+
+Creates a snapshot of the database.
+
+### Synopsis
+
+
+This command creates a logical snapshot of the database and genesis file to initialize a new network. The genesis file is created with the app_hash set to the hash of the snapshot(uncompressed) and the genesis validators set to the validators from the previous network. This command interacts directly with the underlying PostgreSQL server, bypassing any interactions with the kwild node. It requires user with administrative privileges on the database to create snapshot without any restrictions. 
+	
+This command is intended to be used for network migrations to generate exportable snapshots used for initializing a new network, serving as the genesis state. The snapshots generated by this command are different from the ones generated by the snapshot store on running the Kwild instance. The snapshots generated by the snapshot store are used for state sync and are not meant to be used for initializing a new network as they contains state which shouldn't exist at genesis.
+
+```
+kwil-admin snapshot create [flags]
+```
+
+### Examples
+
+```
+# Create database snapshot and the genesis file to initialize a new network
+# Password is optional if the db is operating in a trust authentication mode.
+kwil-admin snapshot create --dbname kwildb --user user1 --password pass1 --host localhost --port 5432 --snapdir /path/to/snapshot/dir
+
+# Snapshot and genesis files will be created in the snapshot directory
+ls /path/to/snapshot/dir
+genesis.json    kwildb-snapshot.sql.gz
+```
+
+### Options
+
+```
+      --dbname string     Name of the database to snapshot (default "kwild")
+  -h, --help              help for create
+      --host string       Host of the database (default "localhost")
+      --password string   Password for the database user (default "password")
+      --port string       Port of the database (default "5432")
+      --snapdir string    Directory to store the snapshot and hash files (default "kwild-snaps")
+      --user string       User with administrative privileges on the database (default "postgres")
+```
+
+### Options inherited from parent commands
+
+```
+      --output string   the format for command output - either 'text' or 'json' (default "text")
+  -S, --silence         Silence logs
+```
```

Comment:
I would personally omit these common flags.  It bloats the docs, but it's fine if y'all want this to be more complete.